website/docs/getting-started.mdx

```diff
@@ -75,15 +128,75 @@ For now choose to use either JSON or Yaml. Below are examples of each that inclu
         version: '0.2'
         dictionaryDefinitions:
             - name: project-words
-            path: './project-words.txt'
-            addWords: true
```

Comment:
Unrelated to other changes in this PR, but went ahead & fixed this while I was in here after I noticed that the yaml example was invalid due to the indention. When used as-is, generates the following error: `Configuration Error: Failed to read config file`.

![image](https://github.com/user-attachments/assets/6e39faa2-3f47-4a3e-8aac-dc0d3099fc15)

Reply:
Thank you.